frontend/src/core/app/permissions.js

```diff
@@ -72,6 +72,48 @@ define(function(require) {
 			
 		},
 
+		hasSuperPermissions:function(){
+			var sessionModelPermissions = Origin.sessionModel.get('permissions');
+			var hasWildCard = false;
+
+			_.each(sessionModelPermissions, function(permission) {
```

Comment:
It's probably a bit more readable if you use `the permissions.hasPermissions` function, and pass `['*/*:delete']` (just specifying delete should be enough?)